<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="id">
     <property name="text">
      <string>id:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tipo">
     <property name="text">
      <string>Tipo sensore:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="scelta">
     <item>
      <property name="text">
       <string>Polveri Sottili</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pressione Atmosferica</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Umidit√†</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raggi UV</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="descr">
     <property name="text">
      <string>Descrizione:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="descrizione"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nome">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nome_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
